include/deal.II/grid/grid_tools.h

```diff
@@ -3156,7 +3156,10 @@ namespace GridTools
   {
     AssertDimension(vertex_indices.size(),
                     GeometryInfo<dim>::vertices_per_cell);
-    return cell_measure<dim>(all_vertices, vertex_indices.data());
+    return cell_measure<dim>(all_vertices,
```

Comment:
I would be fine with this but @bangerth not. I have proposed the same here: https://github.com/dealii/dealii/pull/10429#discussion_r432993866